grader.js

```diff
@@ -506,12 +506,62 @@ var Grader = (function() {
       }
       var hasCorrectStyle = false;
       var currentStyle = elem.css(cssProperty);
-      if (currentStyle  === _correctStyle) {
+      if (currentStyle === _correctStyle) {
         hasCorrectStyle = true;
       }
       return hasCorrectStyle;
     },
 
+    hasStyleOtherThan : function (elem, cssProperty, incorrectStyle) {
+      if (!this.isjQuery(elem)) {
+        elem = $(elem);
+      }
+      var hasStyleOtherThan = false;
+      var currentStyle = elem.css(cssProperty);
+      if (currentStyle != incorrectStyle) {
+      	return true;
+      }
+      return hasStyleOtherThan;
+    },
+
+    isPropertyLessThan : function (elem, cssProperty, value) {
+      if (!this.isjQuery(elem)) {
+        elem = $(elem);
+      }
+      var is = false;
+      if (elem.css(cssProperty) < value) {
+      	is = true;
+      }
+      return is;      	
+    },
+
+    isPropertyGreaterThan : function (elem, cssProperty, value) {
+      if (!this.isjQuery(elem)) {
+        elem = $(elem);
+      }
+      var is = false;
+      if (elem.css(cssProperty) > value) {
+      	is = true;
+      }
+      return is;      	
+    },
+
+    doesPropertyMatch : function (elem, _style, validStyles) {
```

Comment:
Also not sure about this method. What's the use case for this over `hasCorrectStyle`?